integration/shareddata/scalars.go

```diff
@@ -371,3 +373,27 @@ func tigrisSchema(typeString string) string {
 			}`
 	return strings.ReplaceAll(common, "%%type%%", typeString)
 }
+
+func generateBigMap(count int) map[string]any {
+	rand.Seed(time.Now().UnixNano())
+
+	res := make(map[string]any, count)
+
+	for i := 0; i < count; i++ {
+		res[fmt.Sprintf("key-%d", i)] = rand.Int31()
+	}
+
+	return res
+}
+
+// Int32BigAmounts contains int32 values for tests.
```

Comment:
Let's add a comment to explain that this data provider is excluded by default and why (and when it needs to be used).